tests/e2e/Services/Users/UsersBase.php

```diff
@@ -230,6 +230,27 @@ public function testCreateUserSessionHashed(array $data): void
         }
     }
 
+    /**
+     * @depends testCreateUser
+     */
+    public function testCreateToken(array $data): void
+    {
+        /**
+         * Test for SUCCESS
+         */
+        $token = $this->client->call(Client::METHOD_POST, '/users/' . $data['userId'] . '/tokens', array_merge([
+            'x-appwrite-project' => $this->getProject()['$id'],
+        ], $this->getHeaders()), [
+            'expire' => 60,
+        ]);
+
+        $this->assertEquals(201, $token['headers']['status-code']);
+        $this->assertEquals($data['userId'], $token['body']['userId']);
+        $this->assertNotEmpty($token['body']['secret']);
+        $this->assertNotEmpty($token['body']['expire']);
```

Comment:
what are the possible values for the `expires` parameter? Can we have some test cases to test for failure?

Reply:
Added some more test cases